src/main/java/careneighbors/community/Image/Image.java

```diff
@@ -0,0 +1,37 @@
+package careneighbors.community.Image;
+
+
+import careneighbors.community.post.Post;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@ToString
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class Image {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String url;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "post_id")
+    private Post post;
```

Comment:
```suggestion
    @ToString.Exclude
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "post_id")
    private Post post;
```

@ManyToOne 처럼 관계가 맺어진 필드는 ToString메서드 사용시 무한 순환이 발생할 수 있기 때문에 @ToString.Exclude 어노테이션을 붙여 제외시켜야 합니다.